community.license/pom.xml

```diff
@@ -11,7 +11,7 @@
     <groupId>org.kaazing</groupId>
     <artifactId>community.license</artifactId>
 
-    <name>Kaazing Corporation License</name>
+    <name>Community License</name>
     <url>https://github.com/kaazing/community</url>
```

Comment:
We'll have to make a transaction on "enterprise" as well.  What is your recommendation for "name" there?